code/game/objects/items/bodybag.dm

```diff
@@ -334,18 +334,36 @@
 
 /obj/structure/closet/bodybag/cryobag/examine(mob/living/user)
 	. = ..()
-	if(!ishuman(bodybag_occupant))
+	var/mob/living/carbon/human/H = bodybag_occupant 
+	if(!ishuman(H))
 		return
 	if(!hasHUD(user,"medical"))
 		return
 	for(var/datum/data/record/medical_record AS in GLOB.datacore.medical)
-		if(medical_record.fields["name"] != bodybag_occupant.real_name)
+		if(medical_record.fields["name"] != H.real_name)
 			continue
 		if(!(medical_record.fields["last_scan_time"]))
 			. += "<span class = 'deptradio'>No scan report on record</span>"
 		else
 			. += "<span class = 'deptradio'><a href='?src=[text_ref(src)];scanreport=1'>Scan from [medical_record.fields["last_scan_time"]]</a></span>"
 		break
+	if(H.stat != DEAD)
+		return
+	var/timer = 0 // variable for DNR timer check
+	if(H.dead_ticks < TIME_BEFORE_DNR) //Check if DNR timer already passed
+		timer = (TIME_BEFORE_DNR-(H.dead_ticks))*2 //Time to DNR left in seconds
+	if(!H.mind && !H.get_ghost(TRUE)|| H.dead_ticks > TIME_BEFORE_DNR || H.suiciding) //We couldn't find a suitable ghost or patient has passed their DNR timer or suicided, this means the person is not returning
+		. += "<span class = 'deptradio'>Patient is DNR</span>"
+	else if(!H.mind && H.get_ghost(TRUE)) // Ghost is available but outside of the body
+		. += "<span class = 'deptradio'>Defib patient to check departed status</span>"
+		. += "<span class = 'deptradio'>Patient have [timer] seconds left before DNR</span>"
```

Comment:
Use span classes please

Reply:
Sorry dont get what this mean, is it extra_classes or just having it in one line or is itsomething like "<span class='notice' ?

Reply:
Ahh is it the stuff in span.dm?

Reply:
![image](https://github.com/tgstation/TerraGov-Marine-Corps/assets/135410935/155fcfe9-6562-4e69-b6a6-9971d8671477)